components/constellation/constellation.rs

```diff
@@ -1929,15 +1920,15 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
     }
 
     fn handle_get_pipeline(&mut self, frame_id: Option<FrameId>,
-                           resp_chan: IpcSender<Option<(PipelineId, bool)>>) {
+                           resp_chan: IpcSender<Option<PipelineId>>) {
         let frame_id = frame_id.unwrap_or(self.root_frame_id);
         let current_pipeline_id = self.frames.get(&frame_id)
             .map(|frame| frame.current.pipeline_id);
         let current_pipeline_id_loaded = current_pipeline_id
-            .map(|id| (id, true));
+            .map(|id| id);
```

Comment:
Yes, this used to do something, back when it was map(|id| (id, true)). Now it's pointless. Good catch.